src/content/1.7/development/database/_index.md

```diff
@@ -9,6 +9,13 @@ weight: 30
 
 By default, PrestaShop’s database tables start with the `ps_` prefix. This can be customized during installation
 
+{{% notice warning %}}
+**Important**
+
+We strongly recommend to customize your database prefix instead of using the default one.
```

Comment:
Explain why ?

Reply:
I think is not necessary .
But maybe add something like : 
> Changing it will help protect your shop against any attacks (some SQL injection for example) targeting the default table names

Reply:
Perfect

Reply:
Done